luxonis_train/utils/loaders/base_loader.py

```diff
@@ -27,14 +28,55 @@ def __init__(
         self,
         view: str,
         augmentations: Augmentations | None = None,
+        images_name: str | None = None,
     ):
         self.view = view
         self.augmentations = augmentations
+        self._images_name = images_name
+
+    @property
+    def images_name(self) -> str:
```

Comment:
I would rename this to maybe `source_names`. It doesn't have to necessarily be images

Reply:
Also raising an error in case it's `None`.

Reply:
>I would rename this to maybe source_name. It doesn't have to necessarily be images

Or do we always expect this to be an image?

Reply:
The property `images_name` I created due to compatibility with image loggers/visualizers, which expect some of the loader inputs to contain image-like objects which it can process in OpenCV, log as images, etc. Right now, the `config.yaml` owner is responsible for setting this property for their custom multi-input logger so that the rest of the code knows which inputs are images.

Reply:
> Also raising an error in case it's None.
I [moved](https://github.com/luxonis/luxonis-train/commit/1cdcb9f29a365558a74df3c410b1bcf6ce82556d) the declaration of `images_name` to the config, which has `'features'` as the value by default, so unless the user implements their custom loader overriding this property as `None`, it should never be `None`. Is that how you produced this scenario?

Reply:
I see, that makes sense. Though I'd maybe rename it to `image_source` or something like that.